bookkeeper-server/src/test/java/org/apache/bookkeeper/http/TestHttpService.java

```diff
@@ -506,27 +543,34 @@ public void testTriggerAuditService() throws Exception {
         HttpServiceResponse response1 = triggerAuditService.handle(request1);
         assertEquals(HttpServer.StatusCode.NOT_FOUND.getValue(), response1.getStatusCode());
 
-        //2,  POST, should return error for no auditor node.
+        //2,  POST, should success.
+        killBookie(1);
+        Thread.sleep(500);
```

Comment:
what's the sleep for?

Reply:
Thanks. seems not needed. will remove it.